crypto/ec/ec_key.c

```diff
@@ -820,6 +820,13 @@ void EC_KEY_clear_flags(EC_KEY *key, int flags)
     key->dirty_cnt++;
 }
 
+int EC_KEY_decoded_from_explicit_params(const EC_KEY *key)
+{
+    if (key->group == NULL)
```

Comment:
may as well check key == NULL as well if this is a public API..

Reply:
Done